app/bank/domain/account_updater.py

```diff
@@ -0,0 +1,8 @@
+from decimal import Decimal
+from bank.domain.account import Account
+
+
+class AccountUpdater:
```

Comment:
avoid using this

Inside the command handler directly do `account.funds_amount = funds_amount`